src/components/Basic/BorrowTabs/RepayBorrowTab.tsx

```diff
@@ -205,11 +205,7 @@ function RepayBorrowTab({ asset, changeTab, onCancel, setSetting }: Props & Disp
           <>
             <img src={asset.img} alt="asset" />
             <p className="center warning-label">
-              To Repay
-              {' '}
-              {asset.name}
-              {' '}
-              to the Venus Protocol, you need to enable it first.
+              To Repay {asset.name} to the Venus Protocol, you need to enable it first.
```

Comment:
It looks like we are still having issues with eslint / prettier. I am not sure why else this is changing.